content/topics/solo-learn/sql-intermediate/1-data-manipulations/_index.md

```diff
@@ -0,0 +1,23 @@
+---
+content_type: topic
+ready: true
+tags:
+- sql
+- solo-learn
+title: Solo Learn - SQL Intermediate - 1 Data Manipulations
+---

```

Comment:
This intermediate course's first module should have the introductions course's last module as a prerequisite, otherwise the introductory course will not be linked in any way to an active syllabus.